study-builder/tenants/testboston/pages/login.html

```diff
@@ -171,6 +225,11 @@
         // uncomment if you want small buttons for social providers
         // socialButtonStyle: 'small'
     };
+    if (language === 'ht') {
+        // Override where Auth0 loads the default language dictionary so it picks up Creole (ht).
+        realOptions.languageBaseUrl = 'https://storage.googleapis.com/testboston-dev';
```

Comment:
Auth0 Lock does not have built-in support for `ht`. I'm trying to get Lock to load the `ht.js` file from here, but hasn't gotten that working yet. Nonetheless, I think this can be merged up while I work on fixing it.